Model/Config/Backend/Installments.php

```diff
@@ -77,7 +79,7 @@ public function beforeSave()
             $ccTypes = $data['cc_types'];
 
             foreach ($ccTypes as $ccType) {
-                $result[$ccType][$amount] = $installments;
+                $result[$ccType][$amount][$counter++] = $installments;
```

Comment:
Do we need the counter variable here? Or can we just do this instead, to automatically increment the index?

```suggestion
                $result[$ccType][$amount][] = $installments;
```

Reply:
Hey @Morerice, thanks for the review. Indeed, we are not using `counter` variable in anywhere else. I removed it.